data/scenarios/facets.yaml

```diff
@@ -33,3 +33,24 @@ interference_with_protective_measures:
   description: Are there any signs of an attacker trying to maintain their access? (Persistence)
   questions:
     - user_logged_in_win
+
+operating_system:
+	uuid: f131c057-0c06-421d-b606-f3f410927d94
+	display_name: What are the involved operating systems?
+	description: What OperatingSystem(s) is in scope or relevant for this case.
+	questions:
+		- operating_system
+
+time_window_of_interest:
+	uuid: 26ef99ad-fcd8-47c6-9aa7-566dfb2a7e87
+  display_name: Time window of interest facet
+	description: What is the time window that is relevant for this investigation.
+	questions:
+		- time_window_of_interest
+
+external_investigation_reference:
```

Comment:
This will be referenced as an attribute on the sketch and not as a facet.

Reply:
ack will remove